src/api-routes/persons/{osuId}/images.js

```diff
@@ -0,0 +1,42 @@
+import sharp from 'sharp';
+
+import { errorHandler, errorBuilder } from 'errors/errors';
+import { getImageById } from 'db/oracledb/images-dao';
+import { parseQuery } from 'utils/parse-query';
+import { openapi } from 'utils/load-openapi';
+
+const { maximum: maxWidth } = openapi.components.parameters.imageWidth.schema;
+
+/**
+ * Get image by ID
+ *
+ * @type {RequestHandler}
+ */
+const get = async (req, res) => {
+  try {
+    const { osuId } = req.params;
+    const { width } = parseQuery(req.query);
+    const image = await getImageById(osuId);
+
+    if (!image) {
+      return errorBuilder(res, 404, 'No image with the specified OSU ID was not found.');
+    }
+
+    res.contentType('image/jpeg');
+    let result;
+    const sharpImage = sharp(image);
+    if (width) {
+      result = await sharpImage.resize(width, null).toBuffer();
+    } else if (await sharpImage.metadata().width > maxWidth) {
```

Comment:
Will we ever run into this? I think the express-openapi package has validated this parameter and will return a 400 if the input if over the maximum.

Reply:
I added this for the case where someone has an image that is by default greater than the max width. Not sure how much this matters but the size of the response starts to balloon when you get passed 2000 pixel width.